package.json

```diff
@@ -84,14 +87,19 @@
     "jest-junit": "^12.0.0",
     "jquery": "^3.3.1",
     "mocha": "^5.2.0",
+    "moment": "^2.29.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-intl": "^5.8.0",
     "react-redux": "^7.2.2",
+    "react-router": "^5.2.0",
     "react-router-dom": "^5.0.0",
     "react-trigger-change": "^1.0.2",
     "redux": "^4.0.0",
+    "redux-observable": "^1.2.0",
     "regenerator-runtime": "^0.13.3",
+    "rxjs": "^6.6.3",
+    "typescript": "^3.7.0-beta",
```

Comment:
Why do we need to add `typescript` here?

Reply:
@BogdanDenis
Typescript was also indicated as linking warning.

Reply:
@Terala-Priyanka could you attach the warning message? It just seems weird that we need to add typescript that we don't use just to get rid of a warning

Reply:
@BogdanDenis 
![image](https://user-images.githubusercontent.com/104053200/170249298-71afd381-80bb-4d6d-8e24-5b156f2e61f1.png)

Reply:
@BogdanDenis 
Updating to latest version fixes the warning. However, I had added the highest version reported in warning as the package uses that version, in order to avoid any further unforeseen issues.

Reply:
@Terala-Priyanka oh, I mean updating `eslint-plugin-jest` to the latest version, and to remove `typescript` and see if the warning disappears

Reply:
@BogdanDenis I tried that too, it did not remove the warnings.

Reply:
@Terala-Priyanka ok, anyway I think it's better to remove `typescript` dependency and have `eslint-plugin-jest` handle this warning, because it would be confusing to have typescript when we don't use it in the project
Also, according to this comment it's also safe to ignore the warning
https://github.com/jest-community/eslint-plugin-jest/issues/467#issuecomment-572618595

Reply:
@BogdanDenis Thank you very much for the insights. Typescript has been removed from package.json